xwiki-platform-core/xwiki-platform-scheduler/xwiki-platform-scheduler-ui/src/main/resources/XWiki/SchedulerJobSheet.xml

```diff
@@ -39,48 +39,58 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity}}
-#if($doc.getObject('XWiki.SchedulerJobClass'))
-#set($job = $doc.getObject("XWiki.SchedulerJobClass"))
-**$services.localization.render('xe.scheduler.job.name')**
+  <content>{{velocity output='false'}}
+#macro (displayJobProperty $obj $propName)
+  #set ($class = $obj.xWikiClass)
+  #set ($prettyName = $class.get($propName).translatedPrettyName)
+    &lt;dt&gt;
+      &lt;label for="${class.getName()}_${obj.number}_${propName}"&gt;$prettyName&lt;/label&gt;
+    #if ($propName == 'script')
+      &lt;span class='xHint'&gt;
+        $services.localization.render('xe.scheduler.job.scriptexplanation')
+      &lt;/span&gt;
+    #end
+    &lt;/dt&gt;
+    &lt;dd&gt;
+    #if ($propName == 'script' &amp;&amp; $xcontext.action == 'view')
 
-$!doc.display('jobName')
+      {{code language='groovy'}}$!job.getValue($propname){{/code}}
```

Comment:
There was a verification if the script value was empty. Now this is gone and an empty box/code macro is displayed.